bin/ami-provision.sh

```diff
@@ -13,5 +13,9 @@ sudo git clone https://github.com/Automattic/vip-quickstart.git /srv
 # Install Quickstart
 sudo /srv/bin/vip-init --server --domain=vip.dev
 
+mkdir -p /srv/pmc
+
+sudo git clone https://github.com/Penske-Media-Corp/pmc.git /srv/pmc
```

Comment:
This is a private repo, this script may update over time and we might pull and merge, not the best place to add to existing script unless this script can be pull up into Automattic/vip-quickstart